apps/showcase/src/assets/trainings/sdk/steps/plugins/instructions.md

```diff
@@ -0,0 +1,29 @@
+Thanks to the `@ama-sdk/core` plugins, you can intercept the API requests and responses to transform their data or
+perform actions such as authentication, logging, etc.
+
+These plugins will be applied before sending all the API requests and after the reception of the responses.
+
+### Pre-requisite
+- The package `@ama-sdk/core` needs to be installed (which has already been done for you).
+
+### Objective
```

Comment:
I would put the `Objective` section before the prerequisites. It should be one of the first things you read when starting the step.